go.mod

```diff
@@ -59,3 +59,69 @@ require (
 	google.golang.org/grpc v1.44.0
 	gopkg.in/yaml.v2 v2.3.0
 )
+
+require (
+	cloud.google.com/go/bigquery v1.8.0 // indirect
+	github.com/Azure/azure-pipeline-go v0.2.3 // indirect
+	github.com/Azure/azure-storage-blob-go v0.14.0 // indirect
```

Comment:
Unsure of what prompted the 2nd require block here. Was it intended? I didn't see any new imports beyond `runtime` and `unsafe` which I assumed are part of the stdlib.

Reply:
Huh. I didn't think we used `vendor` on this project. I'm lost, but this doesn't feel right to me :shrug:. More research required, but I bet  @jsternberg would have a quick thought on this.

Reply:
We don't use vendor in the flux repository. You would use `go mod tidy` instead.

Reply:
I think this second block is something that was added in go 1.17 as I see it in the influxdb repository.